tests/testthat/_snaps/validate_submission.md

```diff
@@ -1219,7 +1219,7 @@
     Code
       str(dup_model_out_val)
     Output
-      Classes 'hub_validations', 'list'  hidden list of 10
```

Comment:
These snapshots were incorrectly picking up the error from the leaky stub. Now corrected